stdlib/jms/src/main/ballerina/jms/durable_topic_subscriber.bal

```diff
@@ -86,18 +92,18 @@ public type DurableTopicSubscriberEndpointConfiguration record {
     !...
 };
 
-# Caller actions related to durable topic subscriber endpoint
+# Caller actions related to DurableTopicSubscriber endpoint
 public type DurableTopicSubscriberCaller client object {
 
     # Acknowledges a received message
     #
     # + message - JMS message to be acknowledged
-    # + return - error upon failure to acknowledge the received message
+    # + return - Error upon failure to acknowledge the received message
     public remote extern function acknowledge(Message message) returns error?;
 
     # Synchronously receive a message from the JMS provider
     #
-    # + timeoutInMilliSeconds - time to wait until a message is received
-    # + return - Returns a message or nill if the timeout exceededs. Returns an error on jms provider internal error.
+    # + timeoutInMilliSeconds - Time to wait until a message is received
+    # + return - Returns a message or nil if the timeout exceeds, returns an error upon jms provider internal error
```

Comment:
I think two sentences with the full stop was better here. Can we change that?

Change "jms provider" to "JMS Provider" as well.

Reply:
All the other comments are without full stops. To make it consistent all the full stops are removed.